core-bundle/src/Resources/views/Frontend/preview_toolbar_base_js.html.twig

```diff
@@ -0,0 +1,277 @@
+        padding: 5px 6px 6px;
+        border: 1px solid #aaa;
+        border-radius: 2px;
+        background-color: #fff;
+        -moz-appearance: none;
+        -webkit-appearance: none;
+    }
+
+    .cto-toolbar .tl_select {
+        margin: 0;
+        box-sizing: border-box;
+        border: 1px solid #aaa;
+        padding: 5px 22px 6px 6px;
+        border-radius: 2px;
+        background: #fff url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMiIgaGVpZ2h0PSIxMiIgdmlld0JveD0iMCAwIDUwMCA1MDAiPjxsaW5lYXJHcmFkaWVudCBpZD0iYSIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiIHgxPSIxMzEuNTIzIiB5MT0iNDIuNjMiIHgyPSIzNjguNDc4IiB5Mj0iMjc5LjU4NCI+PHN0b3Agb2Zmc2V0PSIwIiBzdG9wLWNvbG9yPSIjYjNiM2IzIi8+PHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjOTk5Ii8+PC9saW5lYXJHcmFkaWVudD48cGF0aCBmaWxsPSJ1cmwoI2EpIiBkPSJNMjUwIDM5Ni42NjZjLTEuMTU1IDAtNC4xMS0xLjgzMi03LjExMy02Ljc1bC0xNjkuNi0yNzcuNDU1Yy0yLjUxNy00LjExNC0zLjE5LTYuOTgtMy4yOC04LjMxNC44MjctLjMzIDIuNTY1LS44MTIgNS42MjctLjgxMmgzNDguNzMzYzMuMDYzIDAgNC43OTguNDgyIDUuNjI3LjgxMi0uMDkgMS4zMzQtLjc2NiA0LjItMy4yOCA4LjMxNWwtMTY5LjYgMjc3LjQ1N2MtMy4wMDUgNC45MTctNS45NiA2Ljc1LTcuMTE0IDYuNzV6Ii8+PC9zdmc+) right -16px center no-repeat;
+        background-origin: content-box;
+        cursor: pointer;
+        text-transform: none;
+        -moz-appearance: none;
+        -webkit-appearance: none;
+    }
+
+    .cto-toolbar .tl_submit {
+        padding: .175em 1em;
+        border: 1px solid #aaa;
+        border-radius: 2px;
+        box-sizing: border-box;
+        cursor: pointer;
+        background: #eee;
+        transition: background .2s ease;
+        font-family: inherit;
+        font-size: 1em;
+        margin-left: 1em;
+    }
+
+    .cto-toolbar .tl_submit:hover {
+        color: inherit;
+        background-color: #f6f6f6;
+    }
+
+    .cto-toolbar .tl_submit:active {
+        color: #aaa;
+    }
+
+    .cto-toolbar__open {
+        display: none;
+        width: 40px;
+        height: 35px;
+        background: #222;
+        border-radius: 0 0 0 2px;
+    }
+
+    .cto-toolbar__open a {
+        display: flex;
+        height: 100%;
+        align-items: center;
+        justify-content: center;
+        opacity: .7;
+    }
+
+    .cto-toolbar__open a:hover {
+        opacity: .9;
+    }
+
+    .cto-toolbar--hidden {
+        border-radius: 0 0 0 4px;
+
+    }
+
+    .cto-toolbar--hidden .cto-toolbar__open {
+        display: block;
+    }
+
+    .cto-toolbar__close {
+        border-left: 1px solid #dddddd;
+    }
+
+    .cto-toolbar__close a {
+        width: 1.5em;
+        font-weight: 600;
+        font-size: 2em;
+        color: #444444;
+        text-decoration: none;
+        display: block;
+        text-align: center;
+        height: 1.5em;
+        line-height: 1.45em;
+    }
+
+    .cto-toolbar__close a:hover {
+        background: rgba(0, 0, 0, 0.03);
+    }
+
+    .cto-toolbar__close a:hover {
+        color: #666666;
+    }
+</style>
+<script>/*<![CDATA[*/
```

Comment:
Why do we need `<![CDATA[` here? Isn't this required for XHTML only?

Reply:
This is adapted from Symfony's toolbar listener. I guess Symfony has to support XHTML while Contao is not using XHTML anymore

Reply:
CDATA removed in 206d929b.